bears/configfiles/TOMLBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='tomlv',
+        output_format='regex',
+        output_regex=r"Error in '(?P<filename>.*)': "
```

Comment:
we have deprecated collecting filenames in regex for non global bears. after all this is going to be executed on a single file right..